2fa/README.md

```diff
@@ -2,8 +2,46 @@
 
 ## What is it?
 
+Two factor authentication (2fa from now on) is a method to double check that you're really you by requiring one more check with something you have. This is usually a code generated by an authenticator application or a physical device, or a text message that's sent to your phone.
+
+When you set up 2fa you provide a special code (seed) of your account / device which will instruct the authenticator to generate the correct codes. This usually happens by scanning a QR code, or by providing the site with a unique identifier of your physical device.
+
+Based on that information, the authenticator will provide a different code that's time based (called a TOTP, or Time based One Time Password). Because only you should have the authenticator set up to your account's seed, other people can't generate the correct code without your device.
+
 ## Why is it important?
 
+Traditionally the way to log into websites, whether it's your bank, Facebook, Twitter, your blog, or any place is to provide a username and password - something that you know. Since there are so many websites now requiring usernames and passwords, and password managers aren't widely used yet, it is a common thing that we reuse passwords across different sites. That also means that if a website got hacked in the past where you used the same username / password as on Facebook, other people could use that information to "guess" your password and gain access to a site that hasn't been compromised yet.
+
+With 2fa turned on, these sites will also require the person logging in to provide another code from something that you should have on your person. This means that the security of those accounts are greatly improved as they will most likely not have access to the device that generates the second code, which means that even though they do have your password, they won't be able to log in.
+
 ## What's happening with it?
 
```

Comment:
Might be worth finding some stats about how many systems do use 2FA (i.e. banks) and what the up-take has been. But, of course, more needs to be done.